### PR TITLE
feat: Python docs

### PR DESCRIPTION
**Pull request by darscan**
Python V1 support docs.

### CODE REVIEW

File: _docs_python/1_support.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: "What type of Python projects does Snyk support?"
+---
+Our Snyk CLI tool supports testing and monitoring Python projects that have their dependencies managed by [pip](https://pypi.python.org/pypi/pip) and expressed in a `requirements.txt` file.
+
+GitHub integration and support for testing projects with Setup.py files is in the works.
```
Comment (adrukh) [inline _docs_python/1_support.md]: I'm in favour of not stating what we do not support yet. Want to make sure this doesn't become a roadmap document :)

File: _docs_python/3_fixing.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Fixing Python projects"
+---
+
+The ability to automatically fix Python vulnerabilities is under development.
```
Comment (adrukh) [inline _docs_python/3_fixing.md]: Same here

File: _docs_python/1_support.md (hunk 1 above)
Comment (darscan) [inline _docs_python/1_support.md]: Cool, will remove

Comment (tkadlec) [inline _docs_python/1_support.md]: Good call @adrukh

**Comment by darscan** [comment]
> We'll need to make sure we also tweak in registry nav

@tkadlec What does this mean?

**Pull request merged**